tesla_ver/bubble_chart/bubble_chart.py

```diff
@@ -245,7 +195,7 @@ def play_pause_switch(n_clicks):
     @app.callback(
         Output("time-slider", "value"), [Input("play-interval", "n_intervals")], [State("time-slider", "value")]
     )
-    def play_increment(n_intervals, time_value):
+    def playIncrement(n_intervals, time_value):
```

Comment:
Function names should be underscored. Only classes use CamelCase

Reply:
addressed in 5a1d5c3